api/src/wfl/api/routes.clj

```diff
@@ -157,15 +165,15 @@
    (merge
     exception/default-handlers
     {;; ex-data with :type :wfl/exception
-     ::workloads/invalid-pipeline          (partial exception-handler 400 "")
-     ::workloads/workload-not-found        (partial exception-handler 404 "")
-     UserException                         (partial exception-handler 400 "")
+     ::workloads/invalid-pipeline          (partial logging-exception-handler 400 "Invalid Pipeline." {:exception-type "InvalidPipeline"} :warning true)
+     ::workloads/workload-not-found        (partial logging-exception-handler 404 "Workload Not Found." {:exception-type "WorkloadNotFound"} :warning true)
+     UserException                         (partial logging-exception-handler 400 "Request Invalid." {:exception-type "UserException"} :warning true)
```

Comment:
A definite improvement over `""`, but …
What do the strings add to the map?
Why `"Invalid Pipeline."` vs `"Request Invalid"`?